core/iwasm/aot/aot_perf_map.c

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
  */
 
+#include <inttypes.h>
```

Comment:
Not include a new header. If required, please add a new definition about `PRIuPTR` in *core/shared/platform/include/platform_common.h*.

Reply:
Can you help me understand why we don't want to include this standard (since C99) header?

Reply:
there was already a PRIxPTR in platform_common, hopefully that does the trick

Reply:
> Can you help me understand why we don't want to include this standard (since C99) header?

platform_common.h includes inttypes.h where available for you.
it also provides fallback definitions of PRIxPTR etc.
i dunno where the fallback is actually necessary though.